_search-plugins/querqy/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Querqy plugin
+has_children: false
+redirect_from:
+  - /search-plugins/querqy/
+nav_order: 10
+---
+
+# Querqy plugin
+
+## Querqy plugin installation
+
+**Querqy is currently only compatible with OpenSearch 1.3.1**
+
+The Querqy plugin code is located [here](https://github.com/querqy/querqy-opensearch). To download the plugin code ZIP file, select the green "Code" button, then select "Download ZIP"
```

Comment:
Suggestion: make the link text more descriptive: (instead of just "here")

[querqy-opensearch] (https://github.com/querqy/querqy-opensearch)

Reply:
made suggested change